database_schemas/cassandra/benchflow.cql

```diff
@@ -135,7 +139,7 @@ CREATE TABLE container_properties (
 );
 
 CREATE TABLE host_properties (
-  host_id uuid,
+  host_id text,
```

Comment:
@VincenzoFerme If the docker one already does what we need, then no reason to have another, it's wasted space in Cassandra.